pkgs/dart_services/lib/src/analysis.dart

```diff
@@ -93,10 +98,10 @@ class AnalysisServerWrapper {
 
     try {
       analysisServer.server.onError.listen((ServerError error) {
-        _logger.severe(
+        _logger.genericSevere(
```

Comment:
This _could_ relate to a request right? Should this stay `severe`?

Reply:
Yes, it could. I do not see how to get the request headers here. Do we build usage telemetry on this?
If yes, we may want to review/adjust it.
If no, it should not matter if it was test call or real call - we want to know about the issue and fix it.

How does it sound?